### ISSUE TITLE
Euphonic Calculations: Performance Benchmarking: Collect results

### ISSUE THREAD

**Issue by rebeccafair** [pace-neutrons/Pace-Project-Plan#62]
Do benchmarking according to and collect results somewhere accessible by everyone (`/work3/projects/pace-neutrons`) on SCARF

Blocked by #59, #61

Sub issue of #26

**Comment by rebeccafair** [comment]
As a result of https://github.com/pace-neutrons/pace-developers/blob/master/euphonic/testing/01_performance_testing_discussion.md Euphonic performance will be routinely tested on SCARF as part of a Jenkins job, so this issue is no longer needed
